From b88858ea420b6df8162230cc4f6b3fb6f960b18b Mon Sep 17 00:00:00 2001
From: Chad Dupuis <cdupuis@redhat.com>
Date: Mon, 18 Aug 2014 19:56:16 -0400
Subject: [scsi] qla2xxx: Use proper message for Non owner reset ACK Timeout

Message-id: <1408391817-6389-29-git-send-email-cdupuis@redhat.com>
Patchwork-id: 87928
O-Subject: [PATCH RHEL 7.1 e-stor 28/69] qla2xxx: Use proper message for Non owner reset ACK Timeout.
Bugzilla: 1089346
RH-Acked-by: Rob Evers <revers@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Atul Deshmukh <atul.deshmukh@qlogic.com>

scsi commit id 145083e624a79d402e3aed366bac9b536811b8c2

Signed-off-by: Atul Deshmukh <atul.deshmukh@qlogic.com>
Signed-off-by: Saurav Kashyap <saurav.kashyap@qlogic.com>
Signed-off-by: James Bottomley <JBottomley@Parallels.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/scsi/qla2xxx/qla_nx2.c b/drivers/scsi/qla2xxx/qla_nx2.c
index b45f4d7..5f88931 100644
--- a/drivers/scsi/qla2xxx/qla_nx2.c
+++ b/drivers/scsi/qla2xxx/qla_nx2.c
@@ -1578,8 +1578,8 @@ qla8044_need_reset_handler(struct scsi_qla_host *vha)
   do {
    if (time_after_eq(jiffies, dev_init_timeout)) {
     ql_log(ql_log_info, vha, 0xb0c4,
-        "%s: Non Reset owner DEV INIT "
-        "TIMEOUT!\n", __func__);
+        "%s: Non Reset owner: Reset Ack Timeout!\n",
+        __func__);
     break;
    }
 
-- 
1.7.1